CDTDatastore/CDTDatastoreManager.h

```diff
@@ -55,9 +55,6 @@ extern NSString *__nonnull const CDTDatastoreErrorDomain;
 
  All datastore files, including attachments and extensions, are deleted.
 
```

Comment:
Should we mention that `[database close]` is now being called? Or, would it be enough in the CHANGELOG.md?